libraries/botframework-connector/src/connectorApi/operations/conversations.ts

```diff
@@ -73,10 +73,11 @@ export class Conversations {
    * example of how to do that would be:
    *
    * ```
-   * var resource = await connector.conversations.CreateConversation(new ConversationParameters(){
-   * Bot = bot, members = new ChannelAccount[] { new ChannelAccount("user1") } );
-   * await connect.Conversations.SendToConversationAsync(resource.Id, new Activity() ... ) ;
-   *
+   * var resource = await connector.conversations.createConversation({
```

Comment:
Thanks for fixing this JS doc string that had C# in it 👍 

One small suggestion on the code, but otherwise this looks fine to me.

```suggestion
   * const resource = await connector.conversations.createConversation({
```